File: test-service-java/src/main/java/com/example/testservicejava/v1/entities/Company.java
package com.example.testservicejava.v1.entities;

import com.fasterxml.jackson.annotation.JsonProperty;
import jakarta.persistence.*;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.EqualsAndHashCode;
import lombok.NoArgsConstructor;

@Entity
@Table(name = "companies")
@Data
@AllArgsConstructor
@NoArgsConstructor
@EqualsAndHashCode
public class Company {

    @Id
    @GeneratedValue(strategy = GenerationType.AUTO)
    @Column(name = "id")
    @JsonProperty(value = "id")
    private Long id;

    @Column(name = "name")
    @JsonProperty(value = "name")
    private String name;

    @Column(name = "bs")
    @JsonProperty(value = "bs")
    private String bs;

    @Column(name = "catch_phrase")
    @JsonProperty(value = "catchPhrase")
    private String catchPhrase;

}

File: test-service-kotlin/src/main/kotlin/org/xapps/service/testservice/v1/entities/Company.kt
package org.xapps.service.testservice.v1.entities

import com.fasterxml.jackson.annotation.JsonProperty
import jakarta.persistence.*

@Entity
@Table(name = "companies")
data class Company(
    @Id
    @GeneratedValue(strategy = GenerationType.AUTO)
    @Column(name = "id")
    @JsonProperty(value = "id")
    var id: Long = 0,

    @Column(name = "name")
    @JsonProperty(value = "name")
    var name: String = "",

    @Column(name = "bs")
    @JsonProperty(value = "bs")
    var bs: String = "",

    @Column(name = "catch_phrase")
    @JsonProperty(value = "catchPhrase")
    var catchPhrase: String = ""
)